app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,35 @@
+<%- if controller_name != 'sessions' %>
+  <div class="login-q">
+    <span>Already have an account?</span>
+    <%= link_to "Log in", new_session_path(resource_name), class: "sign-up-btn"%>
+  </div>
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <div class="login-q">
+    <span>Don't have an account?</span>
+    <%= link_to "Sign up", new_registration_path(resource_name), class: "sign-up-btn"%>
+  </div>
+<% end %>
+
+<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+  <div class="login-q">
+    <%= link_to "Forgot your password?", new_password_path(resource_name) %>
+  </div>
+<% end %>
+
+<%- if devise_mapping.confirmable? && controller_name != 'confirmations'%>
+  <div class="login-q">
+    <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name) %>
+  </div>
+<% end %>
+
+<%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
+  <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.omniauthable? %>
+  <%- resource_class.omniauth_providers.each do |provider| %>
+    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post %><br />
+  <% end %>
+<% end %>
```

Comment:
- [ ] [OPTIONAL] I can't log in if ı am not signup for the website which is excellent. But it will be better if the user can see just one error. Every time when ı click error adding.
![image](https://user-images.githubusercontent.com/38635158/205613875-e1e4a7d1-c1bd-4898-b470-cb725602146e.png)

Reply:
@ahmetbozaci Done, the issue was the condition I added in the `application.html.erb` :heavy_check_mark: